owl/EASE-OBJ.owl

```diff
@@ -2457,45 +2480,6 @@
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Support -->
-
-    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Support">
```

Comment:
There should be a disposition for the ability to support something however. This needs more discussion before it can be committed.

Reply:
There is. I called it *Deposition* as in something to deposit stuff

Reply:
There is, it is called *Deposition*